packages/table/src/Row/Row.tsx

```diff
@@ -8,51 +8,50 @@ import { LGRowData } from '../useLeafyGreenTable';
 
 import InternalRowWithoutRT from './InternalRowWithoutRT';
 import { MemoizedInternalRowWithRT } from './InternalRowWithRT';
-import { RowProps } from './Row.types';
+import { RowComponentType, RowProps } from './Row.types';
 import { RowContextProvider } from './RowContext';
 
 /**
  * Renders the provided cells
  */
-const Row = <T extends LGRowData>({
-  row,
-  virtualRow,
-  disabled = false,
-  ...rest
-}: RowProps<T>) => {
+
+const RowWithRef = <T extends LGRowData>(
+  { row, virtualRow, disabled = false, ...rest }: RowProps<T>,
+  ref: ForwardedRef<HTMLTableRowElement>,
+) => {
   const { theme } = useDarkMode();
-  const { shouldAlternateRowColor } = useTableContext();
+  const { shouldAlternateRowColor = false } = useTableContext();
   const { virtualTable } = useVirtualTableContext();
 
   return (
     <>
       {row ? (
         <MemoizedInternalRowWithRT
-          // @ts-ignore FIXME:
           row={row}
           virtualRow={virtualRow}
           theme={theme}
           measureElement={virtualTable?.measureElement}
-          // @ts-ignore FIXME:
           shouldAlternateRowColor={shouldAlternateRowColor}
           isExpanded={row.getIsExpanded()}
-          // @ts-ignore FIXME:
           isParentExpanded={
-            row.getParentRow() ? row.getParentRow()?.getIsExpanded() : false
+            (row.getParentRow() && row.getParentRow()?.getIsExpanded()) ?? false
           }
           isSelected={row.getIsSelected()}
-          disabled={disabled}
+          ref={ref}
           {...rest}
         />
       ) : (
         <RowContextProvider disabled={disabled}>
-          <InternalRowWithoutRT {...rest} />
+          <InternalRowWithoutRT ref={ref} {...rest} />
         </RowContextProvider>
       )}
     </>
   );
 };
 
+// TODO: Can i avoid asserting?
+export const Row = React.forwardRef(RowWithRef) as RowComponentType;
```

Comment:
In the [SO answer you shared in the type comment](https://stackoverflow.com/a/58473012) the 2nd wrapper option would avoid having to assert here. The 3rd option is quite clean, but I get why we wouldn't want to change the API to some custom ref prop.

Regardless of approach, it might be worth adding a small comment here as well! It's definitely a bit odd at a glance why we're doing this without looking at the type itself to see the comment there. Even if it's just one line summary.